apps/emqx/src/emqx_ws_connection.erl

```diff
@@ -727,7 +726,7 @@ serialize_and_inc_stats_fun(#state{serialize = Serialize}) ->
                     ok = emqx_metrics:inc('delivery.dropped.too_large'),
                     ok = emqx_metrics:inc('delivery.dropped'),
                     <<>>;
-            Data -> ?SLOG(debug, #{msg => "SEND", packet => Packet}),
+            Data -> ?TRACE("SEND", #{transport => websocket}, Packet),
```

Comment:
This, and many other traces, lacks of an “anchor” message (msg) field.
The anchor should be one static word (underscore for white space) with proper namespacing